py/visualize.py

```diff
@@ -486,48 +475,60 @@ def bubble_chart(responses, poll_ids=None, question_ids=None, xtab1_val="-"):
             x="date",
             y="pct_fav",
             color="country",
-            size=size,
-            size_max=size_max,
             opacity=opacity,
             custom_data=custom_data,
             labels=variable_mapping_inverse,
         )
 
-    # add line for zero net fav
-    fig.add_hline(
-        y=0,
-        line_width=3,
-        line_dash="dot",
-        line_color=GRAY,
-        annotation_text="Zero net favorability",
-        annotation_position="bottom left",
-    )
-
     fig.update_layout(
         clickmode="event+select",
         xaxis=dict(
             title=None,  # date self-explanatory
         ),
         legend_title_text=None,  # country self-explanatory
+        autosize=True,
```

Comment:
Could you combine the `update_layout` calls? I think only one `update_traces` call should be necessary too.